lib/commons/builder/legislative_term.rb

```diff
@@ -2,17 +2,17 @@
 
 class LegislativeTerm
   def initialize(legislature:, term_item_id: nil, position_item_id: nil,
-                 start_date: nil, end_date: nil, comment: nil)
+                 start_date: nil, end_date: nil, **options)
     raise 'You must specify a term item or a start and end date' if !term_item_id && !(start_date and end_date)
     @legislature = legislature
     @position_item_id = position_item_id
     @term_item_id = term_item_id
     @start_date = start_date
     @end_date = end_date
-    @comment = comment
+    @extra = options
```

Comment:
OOI, more recent versions of Rubocop insist that you make the instance variable have the same name as the initializer parameter, so you can either pre-empt that now, or leave it to deal with when we upgrade Rubocop :smile:

Reply:
Okay; I'll do that :-)